draft of Artifact rollback variables (#497)
harness/developer-hub: docs/platform/12_Variables-and-Expressions/harness-variables.md

* `<+artifacts.sidecars.[sidecar_identifier].tag>`
* `<+artifacts.sidecars.[sidecar_identifier].connectorRef>`

### Artifact rollback variables

Artifact rollback variable expressions output information on the artifact used for a deployment rollback.

To copy and use the expressions from an executed pipeline, do the following:

1. In a CD pipeline deployment, click **Infrastructure Section**.
2. In **Infrastructure Section**, click the **Output** tab.
3. View the names and values in **Artifact Rollback**.

> 4. Click **Copy** for any **Output Name** to copy the expression.

Reviewer: Per the Microsoft Style Guide, use "Select" instead of "Click".

Author: done. thanks.

> Here's is an example for an ECS deployment using a Docker image in Artifactory:

Reviewer: ```suggestion
The following is an example for an ECS deployment using a Docker image in Artifactory:
```

Reviewer: I'm suggesting "The following" instead of "Here's" for accessibility. "The following" will be more precise for people who use screen readers.

![artifact rollback](static/156302d5dbad242d2faf55205335ec9f152dd13462666bfc197707bfd1d6c393.png)  

For example, if you added a Shell Script step and included both `<+artifact.displayName>` and `<+rollbackArtifact.description>`, you would get the name of the artifact being deployed and the artifact that will be used in case of rollback.

> If `<+artifact.displayName>` gave you `harness/todolist-sample_11_0702794`, then `<+rollbackArtifact.description>` would give you the previous release that will be used in case of rollback: `harness/todolist-sample_10_0702581`.

Reviewer: ```suggestion
If the output of `<+artifact.displayName>` was `harness/todolist-sample_11_0702794`, then the output of `<+rollbackArtifact.description>` would be the previous release that would be used in case of rollback: `harness/todolist-sample_10_0702581`.
```

> Harness pulls rollback artifact information from the last successful deployment. If there's no previous, successful deployment, then the rollback artifact will return null.

Reviewer: ```suggestion
Harness pulls rollback artifact information from the last successful deployment. If there's no previous, successful deployment, then the rollback artifact returns null.
```

> Here is the list of the rollback artifact variables. Review the [artifact variables](#artifact) above for definitions of each variable.

Reviewer: ```suggestion
The following is the list of the rollback artifact variables. Review the [artifact variables](#artifact) for definitions of each variable.
```
